include/aws/io/log_channel.h

```diff
@@ -33,11 +33,11 @@ struct aws_log_writer;
 struct aws_log_channel;
 
 typedef int (*aws_log_channel_send_fn)(struct aws_log_channel *channel, struct aws_string *output);
-typedef int (*aws_log_channel_cleanup_fn)(struct aws_log_channel *channel);
+typedef void (*aws_log_channel_clean_up_fn)(struct aws_log_channel *channel);
```

Comment:
I thought we weren't declaring these as pointers?

Reply:
Nice noticing
I'll fix these up